lib/form_example.dart

```diff
@@ -130,6 +136,11 @@ class _SignUpFormState extends State<SignUpForm> {
           }
           print(
               'sign up with password: "${_passwordController.text}"/"${_confirmPasswordController.text}"');
+          Navigator.push(context, MaterialPageRoute(builder: (_) {
+//!!!!!!!!!!!! Use this button for testing different screens (default: PersonalForm)
```

Comment:
I don't recognize this type of comment.  If you are saying it is temporary, maybe make it a TODO: with the conditions when you would remove it.